pages/login.js

```diff
@@ -0,0 +1,30 @@
+import Image from 'next/image';
+
+import Background from '@/components/Background';
+import LoginForm from '@/components/forms/Login';
+
+import logo from '../public/logo.png';
```

Comment:
Should we use an SVG file for the logo instead of PNG to improve the image quality? 🤔

Reply:
used an svg image in https://github.com/tyrro/react-survey/pull/50/commits/ac569c09a1643adb280fe4d6a3705d1c347ab318